Dockerfile

```
@@ -0,0 +1,98 @@
+
+# =============================================================================
+# Build Stage
+# =============================================================================
+FROM ubuntu:22.04 AS build
+
+ARG BUILD_TYPE
+ARG TARGETPLATFORM
+ARG TARGETARCH
+
+# Install build dependencies
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)